test/tests/functional/pbs_resource_usage_log.py

```diff
@@ -0,0 +1,350 @@
+        )
+        ja.create_eatcpu_job()
+        jid_a = self.server.submit(ja)
+
+        subjid1 = j.create_subjob_id(jid_a, 1)
+        subjid2 = j.create_subjob_id(jid_a, 2)
+
+        self.server.expect(JOB, {'job_state': 'R'}, subjid1)
+        self.server.expect(JOB, {'job_state': 'R'}, subjid2)
+
+        self.assertTrue(self.server.isUp())
+        self.assertTrue(self.mom.isUp())
+
+        # kill -9 mom
+        self.mom.signal('-KILL')
+
+        # Verify that nodes are reported to be down.
+        self.server.expect(NODE, {ATTR_NODE_state: 'down'},
+                           id=self.mom.shortname)
+
+        time.sleep(15)
+
+        self.server.expect(JOB, {ATTR_state: 'Q'}, jid)
+        self.server.expect(JOB, {'job_state': 'Q'}, subjid1)
+        self.server.expect(JOB, {'job_state': 'Q'}, subjid2)
+
+        self.server.tracejob_match(
+            msg='Job requeued, execution node .* down', id=jid,
+            interval=5, regexp=True)
+        self.server.tracejob_match(
+            msg='Job requeued, execution node .* down', id=subjid1,
+            interval=5, regexp=True)
+        self.server.tracejob_match(
+            msg='Job requeued, execution node .* down', id=subjid2,
+            interval=5, regexp=True)
+
+        # now restart mom
+        self.mom.start()
+        self.assertTrue(self.mom.isUp())
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
+        self.server.expect(JOB, {ATTR_substate: '42'}, jid)
+
+        self.server.delete(jid, wait=True)
+        self.server.delete(jid_a, wait=True)
+
+        # job1 has 2 R records and a E record
+        matches = self.server.accounting_match(
+            msg='.*R;' + jid + '.*Exit_status=0.*resources_used.*run_count=1',
+            id=jid, regexp=True, allmatch=True)
+        self.assertEqual(len(matches), 2, " 2 R records expected")
+        self.server.accounting_match(
+            msg='.*E;' + jid +
+            '.*Exit_status=271.*resources_used.*run_count=2',
+            id=jid, regexp=True)
+
+        # job array's subjobs have 2 R records and
+        # the jobarray has E record with run_count=0
+        matches = self.server.accounting_match(
+            msg='.*R;' + re.escape(subjid1) +
+            '.*Exit_status=0.*resources_used.*run_count=1',
+            id=subjid1, regexp=True, allmatch=True)
+        self.assertEqual(len(matches), 2, " 2 R records expected")
+        matches = self.server.accounting_match(
+            msg='.*R;' + re.escape(subjid2) +
+            '.*Exit_status=0.*resources_used.*run_count=1',
+            id=subjid2, regexp=True, allmatch=True)
+        self.assertEqual(len(matches), 2, " 2 R records expected")
+        self.server.accounting_match(
+            msg='.*E;' + re.escape(jid_a) +
+            '.*Exit_status=1.*run_count=0', id=jid_a,
+            regexp=True)
+
+        # Set to defaults
+        a = {ATTR_nodefailrq: 310}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
```

Comment:
PTL should restore this to the default setting.  If it doesn't, then you should fix PTL and not set it here.  If PTL doesn't reset it in setUp(), then if you test fails, this will remain a non-default value.